models.html

```diff
@@ -460,6 +469,142 @@ <h2>Constituency Parsing</h2>
+    These often simplify syntactically complex sentences, and make their predicate-argument structure easily accessible for various downstream tasks.
+    This model is a reimplementation of
+    </span>
+    <a href="https://www.semanticscholar.org/paper/Supervised-Open-Information-Extraction-Stanovsky-Michael/c82921a426fd8090564f459b0bd90cdb1e7a9e2d" target="_blank" rel="noopener noreferrer">a deep BiLSTM sequence prediction model (Stanovsky et al., 2018)</a>.
+  </span>
+
+            </p>
+            <div class="tab">
+              <ul class="tab__nav">
+                <li data-tab="prediction" class="tab__nav__item"><span>Prediction</span>
+                  <div class="tab__nav__item__glyph">
+                    <svg>
+                      <use xlink:href="#icon__disclosure"></use>
+                    </svg>
+                  </div>
+                </li>
+                <li data-tab="evaluation" class="tab__nav__item"><span>Evaluation</span>
+                  <div class="tab__nav__item__glyph">
+                    <svg>
+                      <use xlink:href="#icon__disclosure"></use>
+                    </svg>
+                  </div>
+                </li>
+                <li data-tab="training" class="tab__nav__item"><span>Training</span>
+                  <div class="tab__nav__item__glyph">
+                    <svg>
+                      <use xlink:href="#icon__disclosure"></use>
+                    </svg>
+                  </div>
+                </li>
+                <li class="tab__nav__link">
+                  <a href="http://demo.allennlp.org/open-information-extraction" target="_blank">
+                    <span>View Demo</span>
+                  </a>
+                </li>
+
+              </ul>
+              <div class="tab__page-container">
+                <div data-tab="prediction" class="tab__page">
+                  <div class="tab__page__content"><pre><code class="no-highlight">echo '{"sentence": "John decided to run for office next month."}' > oie-examples.jsonl
+allennlp predict https://s3-us-west-2.amazonaws.com/allennlp/models/openie-model.2018-08-20.tar.gz \
+                        oie-examples.jsonl \
+                        --predictor=open-information-extraction</code></pre></div>
+                </div>
+                <div data-tab="evaluation" class="tab__page">
+                  <div class="tab__page__content">
+                    <p class="t-sm">The Open Information extractor was evaluated on the <a href="https://github.com/gabrielStanovsky/oie-benchmark" target="_blank">OIE2016 corpus</a>.</p>
+                  </div>
+                </div>
+                <div data-tab="training" class="tab__page">
+                  <div class="tab__page__content">
+                    <p class="t-sm">The Open Information extractor was evaluated on the <a href="https://github.com/gabrielStanovsky/oie-benchmark" target="_blank">OIE2016 corpus</a>.</p>
```

Comment:
The data is freely available right? You should upload it to the allennlp s3 bucket and add a config https://github.com/allenai/allennlp/tree/master/training_config here which replicates the training proceedure for this model, if possible.